Explorer/Assets/Scripts/Global/Tests/ParentingSceneShould.cs

```diff
@@ -0,0 +1,116 @@
+using Arch.Core;
+using Cysharp.Threading.Tasks;
+using Global;
+using NUnit.Framework;
+using SceneRunner.Scene;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace ECS.Unity.Systems.Tests
+{
+    /*
+     Using this scene to test parenting. It instantiates 50 cubes, then starts switches the parent on each frame.
```

Comment:
After merging it with my `MeshCollider` branch consider moving the scene to the `StreamingAssets` folder so it will be testable at runtime